### PR TITLE
Remove official support for Python 3.6

### PR DESCRIPTION
**Pull request by gherceg**
https://dimagi-dev.atlassian.net/browse/SAAS-13500

This is the final step in the 3.10 upgrade, which officially drops support for 3.6. While commcare-cloud will still work with 3.6, we are not making any efforts to continue supporting it with future changes.
##### Environments Affected

None
##### Announce New Release

- [x] After merging, I will follow [these instructions](https://confluence.dimagi.com/display/saas/Announcing+changes+affecting+third+parties#Announcingchangesaffectingthirdparties-announcing)
to announce a new commcare-cloud release.

### CODE REVIEW

**Review by joxl — APPROVED** [review]

File: changelog/0067-dropping-3.6-support.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  tools and libraries we are recommending that commcare-cloud run on Python
+  3.10. Python 3.6 has reached end of life and we will no longer make an effort
+  to support it on commcare-cloud.
+
+update_steps: |
+  Follow the steps below to properly install Python 3.10 and its dependencies.
+
+  $ sudo add-apt-repository -y ppa:deadsnakes/ppa
+  $ sudo apt update
+  $ sudo apt-get -y install python3.10 python3.10-dev python3.10-distutils python3.10-venv libffi-dev
```
Comment (millerdev) [inline changelog/0067-dropping-3.6-support.yml:21]:
````suggestion
  ```
  sudo add-apt-repository -y ppa:deadsnakes/ppa
  sudo apt update
  sudo apt-get -y install python3.10 python3.10-dev python3.10-distutils python3.10-venv libffi-dev
  ```
````

File: changelog/0067-dropping-3.6-support.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  Once Python 3.10 is installed on your control machine, run:
+  
+  $ update-code
```
Comment (millerdev) [inline changelog/0067-dropping-3.6-support.yml:25]:
````suggestion
  ```
  update-code
  ```
````

File: changelog/0067-dropping-3.6-support.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  which will pull the latest version of commcare-cloud and trigger the creation and activation of a Python 3.10 virtual environment.
+
+  Confirm the active virtual environment is using Python 3.10
+
+  $ python --version
```
Comment (millerdev) [inline changelog/0067-dropping-3.6-support.yml:31]:
````suggestion
  ```
  python --version
  ```
````

File: changelog/0067-dropping-3.6-support.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+  Finally, run:
+  
+  $ manage-commcare-cloud configure
```
Comment (millerdev) [inline changelog/0067-dropping-3.6-support.yml:35]:
````suggestion
  ```
  manage-commcare-cloud configure
  ```
````

**Review by millerdev — APPROVED** [review]

File: changelog/0067-dropping-3.6-support.yml (hunk 4 above)
Comment (gherceg) [reply changelog/0067-dropping-3.6-support.yml]: See cffa7e21d24a9e6359b65f79796e1fc4dff5af15

File: changelog/0067-dropping-3.6-support.yml (hunk 3 above)
Comment (gherceg) [reply changelog/0067-dropping-3.6-support.yml]: See cffa7e21d24a9e6359b65f79796e1fc4dff5af15

File: changelog/0067-dropping-3.6-support.yml (hunk 2 above)
Comment (gherceg) [reply changelog/0067-dropping-3.6-support.yml]: See cffa7e21d24a9e6359b65f79796e1fc4dff5af15

File: changelog/0067-dropping-3.6-support.yml (hunk 1 above)
Comment (gherceg) [reply changelog/0067-dropping-3.6-support.yml]: See cffa7e21d24a9e6359b65f79796e1fc4dff5af15

**Review by millerdev — APPROVED** [review]

**Comment by gherceg** [comment]
@dimagi/dimagi-dev you should now use Python 3.10 with commcare-cloud. Please upgrade your local environments to use a 3.10 virtualenv.

**Pull request merged**